### DISCUSSION TITLE
Is there a way to set a custom style for when a button is being focused?

### DISCUSSION THREAD

**Discussion by IaKee** [rdbende/Azure-ttk-theme#56] [Question]
I've modified the color scheme of your Azure theme to orange, to fit my application, not much other than that. But I'm having some trouble seeing what is currently being focused, while using it, and by what I've seen so far the default button on tkinter.ttk already has a style set for when the widget is currently being focused (image below), if possible I'd like to use the 'Accent.TButton' style of Azure (the one I'm using on the orange button)  for when a button is being focused (currently using 'focus' as the style does not seem to change its appearance).

![Screenshot_3](https://user-images.githubusercontent.com/44215531/218327964-39f20777-fce4-4c2a-82d8-9ff40623d0e5.png)
![focused button](https://user-images.githubusercontent.com/44215531/218327800-56219665-68ff-4ec4-9942-c6bc3d832568.png)

Thank you for your time!

**Comment by rdbende** [comment]
Hello,
I seem to have missed this completely.

Unfortunately most of the widgets don't have focus indicator with the Azure theme. With #60 they were now added, though only to the buttons.
